geograypher/cameras/cameras.py

```diff
@@ -615,9 +630,9 @@ def get_cameras_in_folder(self, folder: PATH_TYPE):
         subset_cameras = self.get_subset_cameras(imgs_in_folder_inds)
         return subset_cameras
 
-    def get_subset_cameras(self, inds: List[int]):
+    def get_subset_cameras(self, inds: List[int]): 
```

Comment:
Remove trailing whitespace. You can probably configure VSCode to do so. In the future, we might want this to happen automatically.